.travis.yml

```diff
@@ -12,31 +12,37 @@ matrix:
       env:
         - DESC="acceptance tests"
         - CMD="mvn install --projects acceptance-tests --also-make --activate-profiles all --batch-mode --show-version --errors"
+        - SONAR_COMMAND = ""
 
     - jdk: oraclejdk8
       env:
         - DESC="findbugs"
         - CMD="mvn install findbugs:check --projects '!scala-unit-tests,!jmh-scala-tests,!jmh-tests' --activate-profiles all -DskipTests=true --batch-mode --show-version --errors"
+        - SONAR_COMMAND = ""
 
     - jdk: oraclejdk8
       env:
         - DESC="checkstyle"
         - CMD="mvn install checkstyle:check --activate-profiles all -DskipTests=true --batch-mode --show-version --errors"
+        - SONAR_COMMAND = ""
 
     - jdk: oraclejdk8
       env:
         - DESC="unit tests"
         - CMD="mvn install --batch-mode --show-version --errors"
+        - SONAR_COMMAND = "sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=eclipse-collections"
 
     - jdk: oraclejdk8
       env:
         - DESC="compile jmh-tests and performance-tests"
         - CMD="mvn install --projects jmh-tests,performance-tests --also-make --activate-profiles all -DskipTests=true --batch-mode --show-version --errors"
+        - SONAR_COMMAND = ""
 
 script:
-  if [[ "$TRAVIS_EVENT_TYPE" == "cron" && "$DESC" == "unit tests" ]];
-  then mvn install --batch-mode --show-version --errors sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=eclipse-collections ;
-  else eval $CMD;
+  if ["$TRAVIS_EVENT_TYPE" == "cron"]; then
+      eval $CMD $SONAR_COMMAND;
```

Comment:
What happens if you delete all the empty declarations of SONAR_COMMAND = "" but keep the rest as it is?